traffic_ops/testing/api_contract/v4/to_data.json

```diff
@@ -1,6 +1,6 @@
 {
     "user": "admin",
-    "password": "twelve12",
```

Comment:
what the reason to change password?

Reply:
I changed it because twelve is what worked in postman while twelve12 does not.

Reply:
Password varies from environment. Those who're verifying the PR they might/might not use local environment. They will verify via a particular environment with arguments , This file is for testing in local environment, Not a big deal, So no need to change it.